src/plugins/irc-events/link.js

```diff
@@ -163,6 +163,13 @@ function fetch(uri, cb) {
 				// if not image, limit download to 50kb, since we need only meta tags
 				// twitter.com sends opengraph meta tags within ~20kb of data for individual tweets
 				limit = 1024 * 50;
+			} else {
```

Comment:
Can you invert the if above, it will look cleaner.

Reply:
I'm not sure what you're asking me to do, but my interpretation is to make the first if test for whether it is an image, rather than not, and swap the code blocks.

Reply:
Correct.

Reply:
Done.